src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/InnerAssignmentCheck.java

```diff
@@ -57,6 +57,51 @@
  * <pre>
  * &lt;module name=&quot;InnerAssignment"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class MyClass {
+ *
+ *   void foo() {
+ *     int a, b;
+ *     a = b = 5; // violation, assignment to each variable should be in a separate statement
+ *     a = b += 5; // violation
+ *
+ *     a = 5; // OK
+ *     b = 5; // OK
+ *     a = 5; b = 5; // OK
+ *
+ *     double myDouble;
+ *     double[] doubleArray = new double[] {myDouble = 4.5, 15.5}; // violation
+ *     // assignment to 'myDouble' should be in a separate statement
```

Comment:
please remove this comment, just `// violation` is enough

Reply:
Done